public/static/docs/api-reference/get_url.md

```diff
@@ -0,0 +1,32 @@
+# dvc.api.get_url()
+
+Returns the full URL to the <abbr>data artifact</abbr> specified by its `path`
+in a `repo`.
+
+## Signature
+
+```py
+get_url(path, repo=None, rev=None, remote=None)
+```
+
+## Parameters
+
+- `path` - path to the target artifact relative to the repository's root
+
+- `repo` - path or Git URL of a DVC repository
+
+- `rev` - (optional)
+  [Git revision](https://git-scm.com/book/en/v2/Git-Internals-Git-References)
+  (such as a branch name, a tag, or a commit hash). This only works with `repo`
+  URLs.
+
+- `remote` - (optional) name of the [DVC remote](/doc/command-reference/remote)
+  to fetch the target artifact from
+
+## Example
+
+```py
+import dvc.api
+
+resource_url = dvc.api.get_url("data/prepared.tsv", repo="https://github.com/my-org/my-repo.git", remote="my-s3")
+```
```

Comment:
What would the value of `resource_url` look like in this example? I think I should add it below the code block. Cc @Suor

Reply:
I guess I should start actually trying the API myself to answer some of these Q's 😋

Reply:
Tested this with
```py
resource_url = dvc.api.get_url("data/data.xml", repo="https://github.com/iterative/example-get-started")
```
Result: `https://remote.dvc.org/get-started/a3/04afb96060aad90176268345e10355`

Reply:
Updated ref in 68a3dd08